include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -307,6 +307,9 @@ class logging_resource_adaptor final : public device_memory_resource {
  * @return The new logging resource adaptor
  */
 template <typename Upstream>
+[[deprecated(
```

Comment:
Theirs also the `friend` member function in the adaptor class above.

Reply:
Done.